core/src/main/java/org/jboss/hal/core/mbui/form/PropertyFilter.java

```diff
@@ -38,6 +38,11 @@ public boolean test(final Property property) {
                 (p.getValue().hasDefined(REQUIRED) && p.getValue().get(REQUIRED).asBoolean()) ||
                         (p.getValue().hasDefined(NILLABLE) && !p.getValue().get(NILLABLE).asBoolean()));
 
+        // do not include "deprecated" attribute
```

Comment:
I think that's ok. Don't know if we ever need to include a deprecated attribute. If so this wouldn't work.